spyder/plugins/editor/widgets/editor.py

```diff
@@ -2329,6 +2329,14 @@ def __refresh_readonly(self, index):
             if not osp.isfile(finfo.filename):
                 # This is an 'untitledX.py' file (newly created)
                 read_only = False
+            try:

```

Comment:
This try/except should only apply to Windows, right @dalthviz?

Reply:
Yep @ccordoba12 also I think this validation needs to be done only when the file isn't an untitled one